plugins/generator-1.21.1/neoforge-1.21.1/procedures/gui_set_text_textfield.java.ftl

```diff
@@ -1,2 +1,4 @@
-if (guistate.get("text:${field$textfield}") instanceof EditBox _tf)
-	_tf.setValue(${input$text});
\ No newline at end of file
+<#if w.hasElementsOfType("gui")>

```

Comment:
Not sure how this procedure will update the value of the text field, currently it only sets data in the hashmap, however, said value is not set to the actual text field

Reply:
Since you will need to send packets later, it may be better to keep type int as string payload is larger